en/Autosave.md

```diff
@@ -7,20 +7,25 @@ helpCategories: ["General"]
 
 ## Purpose
 
-The autosave feature helps prevent loss of data if your computer or JabRef crashes. 
+The autosave feature helps to prevent loss of data by saving the opened database on user interactions.
+So there is no need to save the database manually after changing it.
+Furthermore it synchronizes your local files which are associated with appropriate [SQLDatabase](shared databases).
```

Comment:
For a user, a shared databases can be either a bib file or a sql database.
To avoid confusion, please, change to "with appropriate [shared SQL databases](SQLDatabase).". (I think the markdown link was in the wrong order).